src/Storages/MergeTree/MergeTreeIndexBloomFilter.cpp

```diff
@@ -781,11 +781,11 @@ bool MergeTreeIndexConditionBloomFilter::traverseTreeEquals(
         if (key_node_function_name == "arrayElement" && (function_name == "equals" || function_name == "notEquals"))
         {
             /** Try to parse arrayElement for mapKeys index.
-              * It is important to ignore keys like column_map['Key'] = '' because if key does not exists in map
+              * It is important to ignore keys like column_map['Key'] = '' because if key does not exist in map
               * we return default value for arrayElement.
```

Comment:
```suggestion
              * It is important to ignore keys like column_map['Key'] = '' because if key does not exist in the map
              * we return default the value for arrayElement.
```